robusta_krr/core/runner.py

```diff
@@ -165,6 +175,10 @@ async def _collect_result(self) -> Result:
                 for obj, (recommended, metrics) in zip(objects, resource_recommendations)
             ],
             description=self._strategy.description,
+            strategy=StrategyData(
```

Comment:
this looks like its supposed to be from a different pr

Reply:
Yeah, I did not checkout accidentally - but it will be removed after another merge